conda_rpms/templates/env.spec.template

```diff
@@ -12,71 +12,39 @@ BuildRoot:      %{_tmppath}/env-{{ env.name }}-label-{{ env.label }}-{{ env.vers
-ln -s {{ install_prefix }}/environments/{{ env.name }}/{{ labelled_tag }} $RPM_BUILD_ROOT{{ install_prefix }}/environments/{{ env.name }}/{{ env.label }}
-mkdir -p $RPM_BUILD_ROOT{{ module_prefix }}/{{ rpm_prefix|lower }}
-cat <<'EOF1' > $RPM_BUILD_ROOT{{ module_prefix }}/{{ rpm_prefix|lower }}/{{ env.name }}-{{ env.label }}
-#%Module1.0
-#
-# scitools - scientific software stack for "{{ env.name }}/{{ env.label }}"
-#
-
-set LINK_DIR "/opt/scitools/environments/{{ env.name }}/{{ env.label }}"
-set BASE_DIR [ exec readlink -f $LINK_DIR ]
-set HELP "Scientific software stack for the '{{ env.name }}' environment and '{{ env.label }}' label."
-
-proc ModulesHelp { } {
-    global LINK_DIR
-    global BASE_DIR
-    global HELP
-
-    puts stderr $HELP
-    puts stderr ""
-    puts stderr "This environment is available via:"
-    puts stderr "   - the symbolic link $LINK_DIR, or"
-    puts stderr "   - the base directory $BASE_DIR"
-    puts stderr ""
-    puts stderr "Note that, the environment symbolic link points to the environment base directory."
-    puts stderr ""
-    puts stderr "As new software is deployed to the scientific software stack, this environment"
-    puts stderr "symbolic link is automatically updated to point to the appropriate environment"
-    puts stderr "base directory."
-}
-
-module-whatis $HELP
-conflict scitools
-prepend-path PATH $BASE_DIR/bin
+mkdir -p $RPM_BUILD_ROOT{{ env.prefix }}/environments/{{ env.name }}
+ln -s {{ env.prefix }}/environments/{{ env.name }}/{{ labelled_tag }} $RPM_BUILD_ROOT{{ env.prefix }}/environments/{{ env.name }}/{{ env.label }}
+{% if module.prefix %}
+mkdir -p $RPM_BUILD_ROOT{{ module.prefix }}
+cat <<'EOF1' > $RPM_BUILD_ROOT{{ module.prefix }}/{{ env.name }}-{{ env.label }}
+{{ module.file }}
 EOF1
-{% if env.name == "default" and env.label == "current" %}
-cat <<'EOF2' > $RPM_BUILD_ROOT{{ module_prefix }}/{{ rpm_prefix|lower }}/.version
-#%Module1.0
-#
-# scitools - scientific software stack default environment
-#
-
-set ModulesVersion "default-current"
+{% if module.default and env.name == module.name and env.label == module.label %}
+cat <<'EOF2' > $RPM_BUILD_ROOT{{ module.prefix }}/.version
+{{ module.default }}
 EOF2
 {% endif %}
+{% endif %}
 
 # This phase just tidies up after itself.
 %clean
 rm -rf $RPM_BUILD_ROOT
 
 %files
 # All files in this directory are owned by this RPM.
-{{ install_prefix }}/environments/{{ env.name }}/{{ env.label }}
-{{ module_prefix }}/{{ rpm_prefix|lower }}/{{ env.name }}-{{ env.label }}
-%dir {{ module_prefix }}/{{ rpm_prefix|lower }}
-{% if env.name == "default" and env.label == "current" %}
-{{ module_prefix }}/{{ rpm_prefix|lower }}/.version
+{{ env.prefix }}/environments/{{ env.name }}/{{ env.label }}
+{% if module.prefix %}
+%dir {{ module.prefix }}
+{{ module.prefix }}/{{ env.name }}-{{ env.label }}
+{% if module.default and env.name == "default" and env.label == "current" %}
```

Comment:
No ideal that the words default and current are in here, but not the subject of this PR.